src/components/table/EditableDraggableTable.css

```diff
@@ -0,0 +1,11 @@
+table.pf-table-inline-edit tbody tr.editing td {
+  background-color: #def3ff; /* color-pf-blue-50 */
```

Comment:
looks like we need scss support in this repo?

Reply:
@vojtechszocs